js/cms.js

```diff
@@ -0,0 +1,131 @@
+// Fetch page data from Strapi
```

Comment:
Just wanted to confirm if we are using strapi or wordpress cms

Reply:
WordPress. I need to clean this file up a bunch.